academy/ibc/clients.md

```diff
@@ -166,7 +166,7 @@ The Tendermint client `ConsensusState` tracks the timestamp of the block being c
 
 <HighlightBox type="tip">
 
-The next validator set is used for verifying subsequent submitted headers or updates to the counterparty `ConsensusState`. See the following part on _Updating clients_ for more information about what happens when a validator set changes between blocks.
+The next validator set is used for verifying subsequent submitted headers or updates to the counterparty `ConsensusState`. See the following part on [Updating clients](https://interchainacademy.cosmos.network/academy/ibc/clients.html#updating-a-client) for more information about what happens when a validator set changes between blocks.
```

Comment:
Why not use `(#updating-a-client)`?

Reply:
note that the link should be put as `./#updating-a-client` per https://v1.vuepress.vuejs.org/guide/markdown.html#:~:text=%5Bfoo%20heading%5D(./%23heading)

Reply:
note that this actually doesn't work 👎 replaced with
`./clients.md#updating-a-client`